### ISSUE TITLE
Page does not work in mobile view

### ISSUE THREAD

**Issue by GGD4V** [MeD-DMC/adapt_authoring_college#8]
https://esdc.prv/cgi-bin/scdtic-ictdcs/page-error-view.aspx?id=14865&wid=0
